website/source/docs/providers/google/r/google_project_iam_policy.html.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: "google"
+page_title: "Google: google_project_iam_policy"
+sidebar_current: "docs-google-project-iam-policy"
+description: |-
+ Allows management of an IAM policy for a Google Cloud Platform project. 
+---
+
+# google\_project\_iam\_policy
+
+Allows creation and management of an IAM policy for an existing Google Cloud
+Platform project.
+
+## Example Usage
+
+```js
+resource "google_project_iam_policy" "project" {
+    project = "your-project-id"
+    policy_data = "${data.google_iam_policy.admin.policy_data}"
+}
+
+data "google_iam_policy" "admin" {
+  binding {
+    role = "roles/editor"
+    members = [
+      "user:jane@example.com",
+    ]
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `project` - (Required) The project ID.
+    Changing this forces a new project to be created.
+
+* `policy_data` - (Optional) The `google_iam_policy` data source that represents
+    the IAM policy that will be applied to the project. The policy will be
+    merged with any existing policy applied to the project.
+
+    Changing this updates the policy.
+
+    Deleting this removes the policy, but leaves the original project policy
+    intact. If there are overlapping `binding` entries between the original
+    project policy and the data source policy, they will be removed.
+
+* `authoritative` - (Optional) A boolean value indicating if this policy
+    should overwrite any existing IAM policy on the project. If this argument
```

Comment:
Let's mention here that this includes removing any IAM policies not specified in your config, and that that could mean you get locked out of your project.